ansible/roles/test/files/ptftests/sflow_test.py

```diff
@@ -213,8 +213,8 @@ def sendTraffic(self):
         ip_dst_addr = '192.168.0.4'
         src_mac = self.dataplane.get_mac(0, 0)
         pktlen=100
-        #send 50*sampling_rate packets in each interface for better  analysis
-        for j in range(0,50,1):
+        #send 100*sampling_rate packets in each interface for better  analysis
+        for j in range(0,100,1):
```

Comment:
Can you add spacing in between for better readability?

Reply:
Done.